databases/week2/part3.sql

```diff
@@ -0,0 +1,43 @@
+USE homework;
+
+-- Task 1 Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT 
+  task.* 
+FROM 
+  task 
+  JOIN user_task ON task.id = user_task.task_id 
+  JOIN user ON user_task.user_id = user.id 
+WHERE 
+  user.email LIKE "%@spotify.com";
+
+-- Task 2 Get all the tasks for 'Donald Duck' with status 'Not started'
+SELECT 
+  task.* 
+FROM 
+  task 
+  JOIN user_task ON task.id = user_task.task_id 
+  JOIN user ON user_task.user_id = user.id 
+  JOIN status ON status.id = task.status_id 
```

Comment:
It works! Another solution could be to simply test for task.status_id = 1, since we know it means not started.